Build/CMake/Modules/GLTFPlatform.cmake

```diff
@@ -38,11 +38,13 @@ elseif (IOS_PLATFORM)
     endif()
 elseif (MSVC)
     if (CMAKE_GENERATOR_PLATFORM STREQUAL "Win32")
-        set(${outPlatform} windows_win32 PARENT_SCOPE)
+        set(${outPlatform} windows_Win32 PARENT_SCOPE)
```

Comment:
If we are now making the platform name (which maps to the output path) use the same case as the CMAKE_GENERATOR_PLATFORM then can we change this to just be the following?

        set(${outPlatform} windows_${CMAKE_GENERATOR_PLATFORM} PARENT_SCOPE)

This will make the output path always match the input "-A" argument case and allow for new platforms to be added without changing this file.

Reply:
Done!